### DISCUSSION TITLE
这地方可真是卧虎藏龙.......

### DISCUSSION THREAD

**Discussion by dominic121-dot** [share121/inter-knot#869] [其他]
![微信图片_20240810175512](https://github.com/user-attachments/assets/acdab8ff-b49a-40d3-a287-f37142f7d77b)
![微信图片_20240810175459](https://github.com/user-attachments/assets/533d192e-79d0-4f29-928d-02cdb0857de6)
你别说这俩还真挺像，我记得剧情里还有个黑帮老爷子也是这个建模吧

**Comment by limbus666** [comment]
是的，那个发委托让咱去教训他们帮里小崽子的那个大爷
